packages/ref-metrics-ilamb3/src/cmip_ref_metrics_ilamb3/example.py

```diff
@@ -0,0 +1,107 @@
+from pathlib import Path
+from typing import Any
+
+import xarray as xr
+from ilamb3.dataset import integrate_space  # type: ignore
+
+from cmip_ref_core.datasets import FacetFilter, SourceDatasetType
+from cmip_ref_core.metrics import DataRequirement, Metric, MetricExecutionDefinition, MetricResult
+
+
+def calculate_global_mean_timeseries(input_files: list[Path]) -> xr.Dataset:
+    """
+    Calculate the global mean timeseries for a dataset.
+
+    Parameters
+    ----------
+    input_files
+        List of input files to calculate the annual mean timeseries.
+
+    Returns
+    -------
+    :
+        The annual mean timeseries of the dataset
+    """
+    ds = xr.open_mfdataset(input_files, combine="by_coords", chunks=None, use_cftime=True)
+    mean: xr.Dataset = integrate_space(ds, "tas", mean=True).to_dataset()
+    return mean
+
+
+def format_cmec_output_bundle(dataset: xr.Dataset) -> dict[str, Any]:
```

Comment:
Do you have any cmec conversion utilities in ilamb3 that could be useful to the REF?

Reply:
I sent a slack message to @acordonez and @minxu74 to see if there is anything generic. I'll also create an longer GH issue when I get around to it to capture some additional requirements.